### PR TITLE
Add support for custom_objects

### LINKED ISSUE #1: In the resumable model  how to call custom_objects.
resumableModel=ResumableModel(model,save_every_epochs=1,to_path='checkpoint.h5')
value error:Unknown loss function:dice_coef_loss
How do make this work?

### PR DESCRIPTION
**Pull request by ViiSkor**

### CODE REVIEW

**Comment by dorukkarinca** [comment]
For reference, this solves https://github.com/dorukkarinca/keras-buoy/issues/1

**Comment by dorukkarinca** [comment]
@ViiSkor I noticed from your repo that you used this project to train on Google Colab. Did it help? Is there room for improvement? I'd appreciate your feedback!

**Comment by ViiSkor** [comment]
> @ViiSkor I noticed from your repo that you used this project to train on Google Colab. Did it help? Is there room for improvement? I'd appreciate your feedback!

Thank you for implementing this library. It accommodated all my needs during my research.
You have to install tf-nightly to make this library work properly on Google Colab.  But the need to install tf-nightly will be eliminated with the next release of TensorFlow, so that's not a big deal.

**Comment by dorukkarinca** [comment]
@ViiSkor If you give me your email address I can add you to AUTHORS.rst

**Comment by ViiSkor** [comment]
@dorukkarinca  vikaskorik@gmail.com
Thx for this useful library one more time :)
